src/net/sourceforge/kolmafia/request/GenericRequest.java

```diff
@@ -1381,16 +1380,7 @@ public void externalExecute() {
   }
 
   public static final boolean shouldIgnore(final GenericRequest request) {
-    String requestURL = GenericRequest.decodeField(request.formURLString);
-    return requestURL == null
-        ||
-        // Disallow mall searches
-        requestURL.contains("mall.php")
-        || requestURL.contains("manageprices.php")
-        || requestURL.contains("backoffice.php")
-        ||
-        // Disallow anything to do with chat
-        request.isChatRequest;
+    return false;
```

Comment:
Interesting. This looks like a personal workspace change, rather than anything to do with Luck!

Reply:
😬